src/Handlebars.Net.Helpers/Helpers/StringHelpers.cs

```diff
@@ -142,6 +142,48 @@ public string Lowercase(string value)
             return value.ToLower();
         }
 
+        [HandlebarsWriter(WriterType.String)]
+        public string PadLeft(string value, object width, string padChar)
```

Comment:
Just use https://docs.microsoft.com/en-us/dotnet/api/system.string.padleft?view=net-5.0

Reply:
So does this mean that you no longer want to have a PadLeft|Right helper? Is it possible to call framework functions that take arguments from directly within templates?

Reply:
Sorry, I misunderstood.

I saw all that code (using if-statements) and I thought you want to build the padleft completely manually.

So, if you can:
1. minimize the if-statements (because I think some logic is already implemented by dotnet itself), so I think you can almost directly call the `return value.PadLeft(totalWidth, padChar[0]);`
2. In case you need some if-statements, please surround these with `{` and `}`

Reply:
Sure thing. I was probably being a little aggressive with guarding against "invalid" input.